tools/cldr-code/src/main/java/org/unicode/cldr/tool/CLDRModify.java

```diff
@@ -2262,8 +2262,8 @@ private boolean wantRevertToBaseline(String xpath, String vxmlValue) {
                     // criterion zero: if vxml value is not ↑↑↑, don't revert to baseline
                     return false;
                 }
-                String baselineValue = baselineFileResolved.getStringValue(xpath);
-                // String baselineValue = baselineFileUnresolved.getStringValue(xpath);
+                // String baselineValue = baselineFileResolved.getStringValue(xpath);
+                String baselineValue = baselineFileUnresolved.getStringValue(xpath);
```

Comment:
this change from resolved to unresolved, in the 2nd commit for this PR, has only small effect -- the only effect seems to be that in a few locales, [draft="contributed"] gets removed from some paths with value ^^^

Reply:
that sounds odd.